go/cmd/config/README.md

```diff
@@ -18,18 +18,19 @@ Terminal B:
 opstrace/packages/app$ yarn console
 ```
 
-Terminal C (requires that a tenant named `tenant-foo` be created):
+Terminal C:
 ```
-opstrace/go/cmd/graphql$ go build && \
+opstrace/go/cmd/config$ go build && \
 GRAPHQL_ENDPOINT=http://127.0.0.1:8080/v1/graphql \
 HASURA_GRAPHQL_ADMIN_SECRET=myadminsecret \
-./graphql \
+./config \
   --loglevel debug \
   --listen "127.0.0.1:8989" \
-  --tenantname tenant-foo \
   --disable-api-authn
 ```
 
+In normal use, the config service extracts the tenant name from the bearer token that must be provided with requests. When instead testing with `--disable-api-authn`, the service requires that we provide the tenant name using an `X-Tenant` header, as provided in the examples below.
```

Comment:
This paragraph makes me think a bit about terminology and concepts, both of which of course are inherently evolving as our product evolves.

Instead of "the bearer token" I so far thought of this as the "Opstrace data API authentication token".

Of course, in colloquial language we will always use shorter versions of this. But, based on experience: instead of using implementation details in the names we call it (such as 'bearer' and 'jwt'), let's maybe say "data API auth token".

That then raises an interesting question: does it make sense to use the data API authentication token as a valid credential against the config API? Probably yes!

Does it make sense to maybe adjust the name "Opstrace data API authentication token" to something else? Probably yes!

"Data API" and "config API" together could be something like "tenant API"?

Maybe Opstrace tenant API authentication token.

A decisive property of the data API authentication tokens that they are _long-lived_ primary credentials.

They have to be, in the trade off space between security and robustness (the observability data ingest path can't be bothered with the hiccups you automatically get by using short-lived authentication tokens).

Does the config API need long-lived authentication tokens? Should the config API maybe, for starters, only be exposed through the UI, involving short-lived secondary credentials?

No need to find answers now.

Reply:
The naming could go against the identity being attested in the token. So if the token just identifies the tenant, then something like "tenant key" or "tenant token" might work. If it also identifies both a user + a tenant the user has access to, it could be a "tenant user key" or "tenant user token"? But I'm not very good with names.